wgpu-info/Cargo.toml

```diff
@@ -3,14 +3,18 @@ name = "wgpu-info"
 version.workspace = true
 authors.workspace = true
 edition.workspace = true
-description = "Adapter information and per-adapter test program"
+description = "A tool to print and process information about available wgpu adapters."
 homepage.workspace = true
 repository.workspace = true
 keywords.workspace = true
 license.workspace = true
-publish = false
 
 [dependencies]
+anyhow.workspace = true
 bitflags.workspace = true
 env_logger.workspace = true
+pico-args.workspace = true
+serde.workspace = true
+serde_json.workspace = true
 wgpu.workspace = true
+wgpu-types = { workspace = true, features = ["serde"] }
```

Comment:
we have a bug in wgpu-types that causes `cargo run -p wgpu-info` not to work standalone: The `serde` feature is on, but because neither `trace` nor `replay` are enabled, neither `serde::Serialize` nor `serde::Deserialize` are active causing errors like this for me:

```
error: cannot find attribute `serde` in this scope
    --> wgpu-types/src/lib.rs:5932:31
     |
5932 | #[cfg_attr(feature = "serde", serde(rename_all = "kebab-case"))]
     |                               ^^^^^
     |
     = note: `serde` is in scope, but it is a crate, not an attribute
```

Not sure if this is a rust version thing (but the toolchain file should enforce same version for all of us) or why you didn't hit this 🤔 
To work around this I had to enable `trace` here locally

Reply:
For reference, this is https://github.com/gfx-rs/wgpu/issues/3609

Reply:
Weirdly I only hit this in my follow up work... I'll backport the changes I made for that.